### ISSUE TITLE
Enhancement regarding: Resource view - horizontal #30

### ISSUE THREAD

**Issue by xfinitysoft** [vkurko/calendar#282]
This enhancement is related to [Issue discussed here](https://github.com/vkurko/calendar/issues/30)

To explain exactly, here is the reference site where it is already implemented: [Single Resource Timetable](https://demo.mobiscroll.com/javascript/timeline/monthly-timetable-vertical-days-horizontal-times#)

As in the v3.0.0. Multiple Resource view is implemented, well is the single resource view available and implemented in the Library?

We look forward to your quicker response.

Thank You

**Comment by vkurko** [comment]
No, this kind of view is not available in Event Calendar. It also seems to me that FullCalendar does not support it. But I think this is an interesting feature. I will consider implementing this kind of view if more users will request it.
